core/monitor/verify.go

```diff
@@ -153,31 +149,28 @@ func (m *Monitor) verifyMutations(muts []*pb.MutationProof, oldRoot, expectedNew
 		if err := merkle.VerifyMapInclusionProof(mapID, index,
 			leaf, oldRoot, mut.GetLeafProof().GetInclusion(), m.mapHasher); err != nil {
 			glog.Infof("VerifyMapInclusionProof(%x): %v", index, err)
-			s, err := status.Newf(codes.DataLoss, "invalid  map inclusion proof: %v", err).WithDetails(mut.GetLeafProof())
-			errList = appendErr(errList, s, err)
+			errs.append(status.Newf(codes.DataLoss, "invalid  map inclusion proof: %v", err).WithDetails(mut.GetLeafProof()))
 		}
 
 		// compute the new leaf
 		newValue, err := mutator.Mutate(oldLeaf, mut.GetMutation())
 		if err != nil {
 			glog.Infof("Mutation did not verify: %v", err)
-			s, err := status.Newf(codes.DataLoss, "invalid mutation: %v", err).WithDetails(mut.GetMutation())
-			errList = appendErr(errList, s, err)
+			errs.append(status.Newf(codes.DataLoss, "invalid mutation: %v", err).WithDetails(mut.GetMutation()))
 		}
 		newLeafnID := storage.NewNodeIDFromPrefixSuffix(index, storage.Suffix{}, m.mapHasher.BitLen())
 		newLeaf, err := entry.ToLeafValue(newValue)
 		if err != nil {
 			glog.Infof("Failed to serialize: %v", err)
-			s, err := status.Newf(codes.DataLoss, "failed to serialize: %v", err).WithDetails(newValue)
-			errList = appendErr(errList, s, err)
+			errs.append(status.Newf(codes.DataLoss, "failed to serialize: %v", err).WithDetails(newValue))
 		}
 
 		// BUG(gdbelvin): Proto serializations are not idempotent.
 		// - Upgrade the hasher to use ObjectHash.
 		// - Use deep compare between the tree and the computed value.
 		newLeafHash, err := m.mapHasher.HashLeaf(mapID, index, newLeaf)
 		if err != nil {
-			errList = append(errList, err)
+			errs = append(errs, err)
```

Comment:
Maybe define in addition a single-arg `func (el errList) append(err)` so this case looks more like the ones above?